website/docs/reference/global-configs/redshift-changes.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Redshift adapter behavior changes"
+id: "redshift-changes"
+sidebar: "Redshift"
+---
+
+## restrict_direct_pg_catalog_access
+
+Originally, the `dbt-redshift` adapter was built on top of the `dbt-postgres` adapter and used Postgres tables for metadata access. With this flag enabled, the adapter will use the Redshift API (through the Python client) if available, or query Redshift's `information_schema` tables instead of using `pg_` tables.
```

Comment:
```suggestion
Originally, the `dbt-redshift` adapter was built on top of the `dbt-postgres` adapter and used Postgres tables for metadata access. When this flag is enabled, the adapter uses the Redshift API (through the Python client) if available, or queries Redshift's `information_schema` tables instead of using the `pg_` tables.
```